test/libsolidity/semanticTests/externalSource/_external/external.sol=sol

```diff
@@ -0,0 +1,5 @@
+contract External {
```

Comment:
Why the `=` in the file name?

Reply:
This is just the test-case `test/libsolidity/semanticTests/externalSource/multiple_equals_signs.sol`. The test just checks the behaviour if multiple equal signs where used  in a remapping. The first equal sign is used for the remapping. The second one gets correctly detected as a part of the filename.